doc/user/data/sql_funcs.yml

```diff
@@ -283,6 +283,9 @@
     description: Encode `b` using the specified textual representation.
     url: encode
 
+  - signature: 'get_byte(b: bytea, n: int) -> int'
+    description: Return the `n`th byte from `b`.
```

Comment:
Nit, but is it clear which way one counts bytes? Least significant, or most significant first? Not hard to find out, for sure, but if it is trivial to state that would rule!

Reply:
Added a description to express that they're counted string-like, left to right.